projects/templates/projects/add_publications.html

```diff
@@ -37,11 +37,23 @@ <h2>
 </div>
 
 <form method="post">
-        {% csrf_token %}
-        {% bootstrap_form pubs_form %}
-        <div class="form-group">
-          <button type="submit" class="btn btn-success">Add</button>
-        </div>
-      </form>
+  {% csrf_token %}
+
+  <h4>Publications BibTeX Format:</h4>
+  <p>Please enter your publication(s) in BibTeX format in the field below. If available, include a link to the publication using the <code>note</code> or <code>howpublished</code> field with the <a href="https://ctan.org/pkg/url">url package</a>.</p>
+  {% bootstrap_form pubs_form %}
+  <h4>Guidelines for Submission:</h4>
+  <ul>
+    <li>Please ensure that the publications you upload are directly related to research or projects that have utilized Chameleon resources.</li>
+    <li>Avoid duplicate submissions by checking if your publication is already listed <a href="https://chameleoncloud.org/user/projects/chameleon-used-research/">here.</a></li>
```

Comment:
This link should also have `taget="_blank"` so we don't interrupt the form they're filling out